index.md

```diff
@@ -2,9 +2,13 @@
 marp: true
 theme: default
 paginate: true
-header: ![height:40px](https://quantstack.net/img/logo.svg)
```

Comment:
Removing it for now to save some vertical space.

Maybe we could add to the footer instead.